src/main/java/com/mail/entities/TextMessage.java

```diff
@@ -0,0 +1,37 @@
+package com.mail.entities;
+
+
+import lombok.*;
+import lombok.experimental.FieldDefaults;
+
+import javax.persistence.*;
+import java.time.LocalDate;
+
+@FieldDefaults(level = AccessLevel.PRIVATE)
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+@Entity
+@Table(name = "messages")
+public class TextMessage {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(unique = true, nullable = false)
+    int id;
+
+    @Column(nullable = false)
+    String title;
+
+    @Column(nullable = false)
+    String from_who;
+    @Column(nullable = false)
```

Comment:
Keep single empty line after each classfield when annotations are present.
This way it is easier to read the code.